mailpoet/lib/Automation/Integrations/WooCommerce/Subjects/CustomerSubject.php

```diff
@@ -47,8 +47,12 @@ public function getArgsSchema(): ObjectSchema {
   public function getPayload(SubjectData $subjectData): Payload {
     $customerId = $subjectData->getArgs()['customer_id'];
     $orderId = $subjectData->getArgs()['order_id'] ?? null;
+
+    $order = $orderId === null ? null : wc_get_order($orderId);
+    $order = $order instanceof WC_Order ? $order : null;
+
     if (!$customerId) {
-      return new CustomerPayload(null, $orderId);
+      return new CustomerPayload(null, $order);
```

Comment:
Why wasn't this catched by phpstan?

Reply:
@websupporter I guess `$subjectData->getArgs()['order_id']` has no type (sth like `mixed`).

Reply:
Do you think it makes sense to make our tests stricter (or some other method) so this type error would be catched in the future?

Reply:
@websupporter Do you mean this particular issue or some other more generic solution? I think in this case, casting the `$orderId` to to `(int)` will help.

Reply:
@websupporter https://github.com/mailpoet/mailpoet/pull/5534/commits/e0d9a07b29b964da93e1188a969a6f15b540e182

Reply:
LGTM. I meant to make phpstan policies more strict to warn in cases of `mixed`. I could imagine this is a slightly bigger task. Maybe a good janitor task.